contracts/test/SSVNetwork_v2.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.16;
+
+import "../SSVNetwork.sol";
+
+contract SSVNetwork_v2 is SSVNetwork {
+    
+    function initializev2(uint32 validatorsPerOperatorLimit_) reinitializer(2) external {
```

Comment:
i suggest making a var called version and instead of hard code 2 to show version + 1 so user better understands how to handle this but either way is fine

Reply:
cool, let me merge first the _version function_ PR and adapt this test to make it coherent as you suggest, thanks!